app/models/trip.rb

```diff
@@ -0,0 +1,8 @@
+class Trip < ApplicationRecord
+  belongs_to :passenger
+  belongs_to :driver
+
```

Comment:
You don't need to write validations for the foreign key relationships.  Rails does that by default.  

You could also add validations to limit the values that the rating could be, and to limit the cost to be greater than 0.